src/com/facebook/buck/apple/AppleDescriptions.java

```diff
@@ -488,9 +488,15 @@ static AppleBundle createAppleBundle(
         params.getBuildTarget(),
         MultiarchFileInfos.create(appleCxxPlatforms, params.getBuildTarget()));
 
-    AppleBundleDestinations destinations =
-        AppleBundleDestinations.platformDestinations(
-            appleCxxPlatform.getAppleSdk().getApplePlatform());
+    AppleBundleDestinations destinations = null;
```

Comment:
I'm not sure if this is the best way to do this in java. Style comments appreciated.